doc/sphinx/documentation/tutorials/small_molecule_tutorial.rst

```diff
@@ -111,9 +104,72 @@ At this stage the reflections may be integrated.  This is done by running
 
 .. code-block:: bash
 
-   dials.integrate refined.refl refined.expt nproc=8
+   dials.integrate refined.refl refined.expt
+
+which will integrate each sequence in sequence, again using all available cores. After integration you can look at the integration shoeboxes with the image viewer, as 
+
+.. code-block:: bash
+  
+  dials.reciprocal_lattice_viewer integrated.refl integrated.expt
+
+
+
+Symmetry Determination
+----------------------
+
+So far the data were processed with a triclinic unit cell, which is usually OK but not ideal for correctly _scaling_ the data. After integration it is usual to determine the symmetry for scaling using
+
+.. code-block:: bash
+
+  dials.symmetry integrated.refl integrated.expt
+
+This will look at the shape of the unit cell and determine the maximum possible symmetry based on the cell parameters, with some tolerance. Each of the possible symmetry operations will be individually tested and scored, and those operations identified as being present will be composed into the point group to be assigned to the data. An attempt is then made to estimate the space group from the presence or absence of axial reflections: this is rather less reliable than the point group determination but also less important for the scaling. After the point group has been determined the reflections will be reindexed automatically to match the correct setting, ensuring that the data are correctly prepared for scaling. 
+
+
+Scaling
+-------
+
+
+In general there is very little which needs to be adjusted in the scaling, as the process is largely automatic. The default model to use for the scaling corrections is "physical" which defines smoothly varying scale factors for the overall intensity and sample decay, with a smoothly varying surface expressed with four or six orders of spherical harmonics for the absorption correction. For organic small molecule crystals the absorption is unlikely to be substantial, but if metals are present or the data are taken with a long wavelength it may be helpful to reduce the strength of the restraints on the absorption surface by setting the absorption level
```

Comment:
Is it worth adding a comment re: Profile fitting vs summation here? As `dials.scale` ignores the profile fitted data entirely:
```
Performing multi-dataset profile/summation intensity optimisation.
+-----------------+---------+---------+
| Combination     |   CC1/2 |   Rmeas |
|-----------------+---------+---------|
| prf only        | 0.9999  | 0.02849 |
| sum only        | 0.99989 | 0.02838 |
| Imid = 2501.27  | 0.99907 | 0.07399 |
| Imid = 45426.39 | 0.99161 | 0.16232 |
| Imid = 4542.64  | 0.99831 | 0.09249 |
| Imid = 454.26   | 0.99979 | 0.038   |
+-----------------+---------+---------+
Summation intensities determined to be best for scaling.
```

Reply:
Well spotted, I had not noticed that 🤔

Reply:
@jbeilstenedmands I would welcome your opinion here. This is an interesting observation. Reviewed and verified this myself.

Reply:
It is _very odd_ that the fully prf and fully sum data are ~ equivalent but any weighted average is _much worse_

Reply:
Does look unusual. This intensity optimisation is done fairly early on in the scaling process, so there could be a few outliers here skewing the results. If you rescale again starting from `scaled.{expt,refl}`, is the same result found? I'll try working through the tutorial myself later this afternoon. Also if you give say `Imid=450` on the command line, are the final merging stats better or worse?

Reply:
Run on the pre-scaled data 

```
Performing multi-dataset profile/summation intensity optimisation.
+-----------------+---------+---------+
| Combination     |   CC1/2 |   Rmeas |
|-----------------+---------+---------|
| prf only        | 0.99932 | 0.03473 |
| sum only        | 0.99944 | 0.03257 |
| Imid = 2417.15  | 0.99851 | 0.07993 |
| Imid = 39328.48 | 0.99055 | 0.1714  |
| Imid = 3932.85  | 0.99788 | 0.0956  |
| Imid = 393.28   | 0.99936 | 0.04177 |
+-----------------+---------+---------+
Summation intensities determined to be best for scaling. 
```

With `Imid=450` the merging stats get a _teeny_ bit worse at low angle and somewhat better (as may be expected) at high

Reply:
This looks like a genuine bug, so I've created an issue for it: https://github.com/dials/dials/issues/1927
Let's add further discussion there, but in terms of the final output results, the difference between summation intensities and using a mid-point crossover is minimal, so I don't think this is a blocker for this tutorial rewrite. Thanks all

Reply:
@huwjenkins looks like you spotted a real bug -> 

https://github.com/dials/dials/issues/1927

Are you happy that we resolve the conversation here?